mne/preprocessing/tests/data/test_eeg_csd.csv

```diff
@@ -0,0 +1,64 @@
65e-07,-9.535707070707071393e-07,-1.782122535353536263e-06,-1.878366161616162313e-06,-1.873111111111111534e-06,-1.932398080808081015e-06,-2.294011058585859297e-06,-1.901767797979797650e-06,-1.334217878787879261e-06,-2.737779494949494935e-06,-2.401571171717171087e-06,-1.174745909090909443e-06,-1.830920202020202720e-06,-1.781773696969696989e-06,-1.878401717171715674e-06,-1.674328888888888843e-06,-3.405469505050504410e-06,-1.916900404040403897e-06,-2.297948787878788742e-06,-2.541635151515151118e-06,-2.056207171717171119e-06,-1.493090808080807635e-06,-1.275531555555555268e-06,-1.459067979797979922e-06,-1.289223777777776823e-06,-1.582193282828282658e-06,-6.926108484848482352e-07,-6.551777777777775786e-07,-1.844046666666666732e-06,-1.158395050505050846e-06,-8.172677777777778342e-07,-4.754168080808081803e-07,-1.106349494949495421e-06,-7.092489898989899087e-07,-4.087575757575759514e-07,-3.173779797979800266e-07,-5.444078686868690600e-07,1.298939393939391704e-07,-1.971121212121221307e-08,1.546825252525252248e-07,-4.556338888888891659e-07,3.620498181818180749e-07,1.505070707070699127e-08,-3.632755555555557054e-07,4.111704040404040324e-07,-3.848647474747474426e-07,1.605539393939393139e-07,7.467154545454546276e-07,-1.551380808081010630e-09,-6.701411111111108149e-07,4.782297979797979697e-07,9.372864646464648976e-07,1.810063030303029613e-07,-1.657461616161615004e-07,-3.878861616161613177e-07,-1.222157343434343755e-06,-4.253707070707070021e-07,-4.850050101010101389e-07,-1.144161252525251733e-06,-3.663617171717169125e-07,-1.399158888888888266e-06,-1.273428080808080825e-06,-1.470858383838383987e-06,-1.411206748848485242e-06,-4.255845454545451958e-07,-4.925266666666663932e-07,-1.306283888888888965e-06,-3.062626262626244851e-08,-1.148840404040401664e-07,-5.489633333333334388e-07,3.663023232323229267e-07,5.390363636363642629e-08,-5.524502020201985514e-08,-3.029441848484847305e-07,-2.607717171717170900e-07,-4.825616161616158453e-08,-2.538674747474747907e-07,4.101132323232323078e-07,2.658551616161619900e-07,8.892952020202020153e-07,-2.229958585858585517e-07,3.693985858585853795e-07,2.057048090909090783e-07,6.391091616161618726e-07,4.053931515151513576e-07,-1.009909393939393781e-06,-4.271014141414098877e-08,4.522194444444443320e-07,1.714669696969694697e-07,-2.702303131313129525e-07,-6.859908686868686411e-07,-3.714364242424240003e-07,1.414630888888890034e-07,-8.757466060606058561e-07,-8.691700000000002053e-07,-9.448299999999993886e-07,-6.156907777777778286e-07,-7.489006161616161631e-07,-4.194075050505051570e-07,-1.208135555555555185e-06,-1.170046677777777669e-06,-1.325560434343433855e-06,-1.229559535353535155e-06,-1.506960131313131181e-06,-1.228605131313130989e-06,-2.758265959595958823e-07,-7.359676868686865129e-07,-7.129717171717171820e-07,-5.529046464646465404e-07,-1.052502424242424204e-06,-1.641118696969696343e-06,-1.781973242424242774e-06,-1.361436161616160979e-06,-2.032656161616161136e-06,-1.681479292929292518e-06,-1.770453131313130879e-06,-1.318636767676767499e-06
```

Comment:
Since this PR is going to require adding data to `mne-testing-data` anyway, let's not add these files here but in `mne-testing-data` instead